File: car-pooling-rest-service/src/main/java/com/lit/carpooling/rest/model/Person.java
package com.lit.carpooling.rest.model;

public class Person {
    private int id;
    private String firstName;
    private String lastName;
    private String email;
    private String phoneNumber;
    private String postcode;
    private String password;

    public Person(int id, String firstName, String lastName, String email, String phoneNumber, String postcode, String password) {
      this.id = id;
      this.firstName = firstName;
      this.lastName = lastName;
      this.email = email;
      this.phoneNumber = phoneNumber;
      this.postcode = postcode;
      this.password = password;
    }
    public Person(){

    }
    public int getId() {
      return id;
    }

    public void setId(int id) {
      this.id = id;
    }

    public String getFirstName() {
      return firstName;
    }

    public void setFirstName(String firstName) {
      this.firstName = firstName;
    }

    public String getLastName() {
      return lastName;
    }

    public void setLastName(String lastName) {
      this.lastName = lastName;
    }

    public String getEmail() {
      return email;
    }

    public void setEmail(String email) {
      this.email = email;
    }

    public String getPhoneNumber() {
      return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
      this.phoneNumber = phoneNumber;
    }

    public String getPostcode() {
      return postcode;
    }

    public void setPostcode(String postcode) {
      this.postcode = postcode;
    }

    public String getPassword() {
      return password;
    }

    public void setPassword(String password) {
      this.password = password;
    }
}

File: src/app/model/person.ts

export class Person{
  id: number;
  firstName: string;
  lastName: string;
  email: string;
  phoneNumber: string;
  postcode: string;
  personPassword: string;

  constructor(FirstName: string, LastName: string,  Email: string, PhoneNumber: string, Postcode: string, PersonPassword: string) {
    this.id = 0;
    this.firstName = FirstName;
    this.lastName = LastName;
    this.email = Email;
    this.phoneNumber = PhoneNumber;
    this.postcode = Postcode;
    this.personPassword = PersonPassword;
  }
}

